qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -60,6 +68,36 @@ def test_importable_types(self):
 
     # TODO: add tests for type/directory/transformer registrations
 
+    def test_importable_formats(self):
+        obs = self.pm.importable_formats
+        exp = {
+            'IntSequenceDirectoryFormat': IntSequenceDirectoryFormat,
+            'MappingDirectoryFormat': MappingDirectoryFormat,
+            'IntSequenceV2DirectoryFormat': IntSequenceV2DirectoryFormat,
+            'IntSequenceFormatV2': IntSequenceFormatV2,
+            'FourIntsDirectoryFormat': FourIntsDirectoryFormat,
+            'IntSequenceFormat': IntSequenceFormat
+        }
+        self.assertEqual(set(obs), set(exp))
+
+    def test_importable_formats_with_dummy(self):
+        obs = self.pm.importable_formats
+        self.assertFalse('UnimportableFormat' in obs.keys())
+        self.assertFalse(UnimportableFormat in obs.values())
+        self.assertFalse('UnimportableDirectoryFormat' in obs.keys())
+        self.assertFalse(UnimportableDirectoryFormat in obs.values())
+
+        obs = self.pm.formats.items()
+        exp = {
```

Comment:
It's probably enough to just assert that the format names exist in `.formats` keys:

```python
self.assertIn('UnimportableFormat', obs)
self.assertIn('UnimportableDirectoryFormat', obs)
```